modules/service/src/main/scala/lucuma/odb/service/ExecutionEventService.scala

```diff
@@ -171,6 +172,16 @@ object ExecutionEventService {
         command: SequenceCommand
       )(using Transaction[F], Services.ServiceAccess): F[Result[ExecutionEvent]] = {
 
+        extension (s: SequenceCommand)
+          def isTerminal: Boolean =
+            s match {
+              case SequenceCommand.Abort    |
+                   SequenceCommand.Stop       => true
+              case SequenceCommand.Continue |
+                   SequenceCommand.Pause    |
+                   SequenceCommand.Start      => false
+            }
+
```

Comment:
I suppose this should move into core at some point.

Reply:
Yes thanks.  I'll do it now and wait for the next core update to take advantage of it.